scripts/run

```diff
@@ -1,9 +1,22 @@
 #!/usr/bin/env bash
-set -ex
+set -e
 
-docker build --tag barman_metrics .
+if [ ! -f config.json ]; then
+    echo "Config file not found."
+    echo "Please create a config.json file in the same format as the config-example.json file in this repo."
+    exit -1;
+fi
+
+HERE=${BASH_SOURCE%/*}
+$HERE/extract-config
+
+source config.sh
+
+docker build --tag $tag .
```

Comment:
Can we build the image on teamcity like the other images, push out to docker hub and our local registry? Or we can move this into a future ticket...